tests/DetailsComponent.js

```diff
@@ -0,0 +1,2 @@
+description: "h1",
+goBackLink: "#go-back"
```

Comment:
> For links, you can use a [special selector](https://github.com/sgmap/mes-aides-ui/blob/master/test/integration/components/HomepageComponent.js).
> Here, it could be `goBackLink: { a: 'Voir' },`
> This would avoid introducing an id.